electronic_straw_polls.bs

```diff
@@ -243,29 +245,33 @@ We propose that subgroup chairs should have the option to conduct subgroup
 
 Henceforth, [=straw polls=] should be divided into two categories:
 
-* <dfn>Direction Straw Polls</dfn>: These polls are taken to suggest the
-      direction that a proposal should pursue in the next revision that comes
-      before the subgroup taking the poll.
+* <dfn>Direction Straw Polls</dfn>: These polls are taken to reaffirm a proposal's
+      direction, or suggest direction changes that a proposal should pursue in the
+      next revision that comes before the subgroup taking the poll.
     These polls will usually be taken on younger proposals, where there are still
       many open questions and areas for additional investigation.
     Direction [=straw polls=] can only be taken on proposals that will return to
       the same subgroup before proceeding further.
+    Dissent should be recorded, so that authors know what work needs to be performed
+      to increase consensus in the next revision of their proposal.
     The results of these polls can always be revisited if new information or new
       perspectives are discovered.
 * <dfn>Decision Straw Polls</dfn>: These polls are taken to make decisions
       regarding a proposal once the subgroup is confident that all applicable
       and relevant aspects of the decision have been discussed.
     These polls will typically be taken on proposals that have been in the
-      subgroup for a few revisions.
+      subgroup for a few revisions, but can also be taken when a proposal is an
+      obvious one to approve, or when a proposal clearly has no chance of gaining
+      consensus and should not be pursued further.
     Decision [=straw polls=] may forward proposals to other subgroups or to
-      plenary.
+      plenary, as well as halt the progress of a proposal.
     The results of these polls can always be revisited if new information or new
       perspectives are discovered.
 
 We propose that the following guidelines should be followed when determining
   whether a poll should be taken synchronously or asynchronously:
 
-* In Library Evolution, Library, Evolution, or Core, at the chair's discretion
+* In Evolution, Core, Library Evolution, or Library, at the chair's discretion
```

Comment:
:p I'll let you get away with going first!

Reply:
You're changing the traditional order, without a reason... this paper should avoid rocking the boat, especially for silly stuff like this.